libraries/botframework-connector/src/teams/teamsConnectorClientContext.ts

```diff
@@ -12,6 +12,12 @@ import {
 } from '@azure/ms-rest-js';
 import { TeamsConnectorClientOptions } from './models';
 
+/**
+ * The Bot Connector REST API extension for Microsoft Teams allows your
+ * bot to perform extended operations on to Microsoft Teams channel
```

Comment:
Maybe?

```suggestion
 * bot to perform extended operations on the Microsoft Teams channel
```

Reply:
Yea, you are right, for some reason my brain read it  as `operations onto Microsoft Teams channel` 😂